publish.js

```diff
@@ -220,6 +220,11 @@ function generate(type, title, docs, filename, resolveLinks) {
     html = helper.resolveLinks(html); // turn {@link foo} into <a href="foodoc.html">foo</a>
   }
 
+  if (env.conf.templates.postProcess
+      && typeof env.conf.templates.postProcess === 'function') {
```

Comment:
Shouldn't need to check that `postProcess` exists here. It being of type `function` necessitates it existing.

```suggestion
  if (typeof env.conf.templates.postProcess === 'function') {
```